packages/ui-library/src/components/input-field-text/index.ts

```diff
@@ -89,8 +93,9 @@ export class BlrInputFieldText extends LitElementCustom {
   };
 
   protected handleChange = (event: Event) => {
+    const inputValue = this._inputFieldTextNode.value;
     if (!this.disabled) {
-      this.dispatchEvent(createBlrTextValueChangeEvent({ originalEvent: event }));
+      this.dispatchEvent(createBlrTextValueChangeEvent({ originalEvent: event, inputValue }));
```

Comment:
Can it be like ({ originalEvent: event, inputValue: this._inputFieldTextNode.value }) so that every single input does not have to create a inputValue variable also it is not used anywhere else.